packages/api-browser/bundle.js

```diff
@@ -0,0 +1,13 @@
+// Need to import using this syntax as require doesn't work for local files + rollup
+import * as browserApi from './build/es/index.js';
```

Comment:
Despite the comment, I still don't quite understand this bit.

This is a JavaScript file importing the built output from TypeScript, but why can't it be `bundle.ts` and import `index.ts` directly? i.e.
```javascript
import * as browserApi from './index';
```

e.g. Does `tsconfig.json` need to be changed to include `./bundle.ts` instead of `./index.ts`? Or does this file need to be JavaScript (i.e. TypeScript doesn't like us messing about with globals)?